src/react/utils/useHydrateAtoms.ts

```diff
@@ -3,12 +3,12 @@ import type { WritableAtom } from '../../vanilla.ts'
 
 type Store = ReturnType<typeof useStore>
 type Options = Parameters<typeof useStore>[0]
-type AnyWritableAtom = WritableAtom<unknown, any[], any>
+export type AnyWritableAtom = WritableAtom<unknown, any[], any>
```

Comment:
I found myself duplicating this type, I feel it would be useful to export it for external usage.

Reply:
Makes sense, it's not a big that I have to duplicate this type.